WalletWasabi/WabiSabi/Client/Batching/Payment.cs

```diff
@@ -0,0 +1,10 @@
+using WalletWasabi.Extensions;
+using NBitcoin;
+
+namespace WalletWasabi.WabiSabi.Client.Batching;
+
+public abstract record Payment(IDestination Destination, Money Amount)
```

Comment:
I don't like those things because while `Payment` is clearly a business model term,  `Base`, `Abstract` and similar are not. They are hits for OOP devs who can infer that it is the root class in a hierarchy. I prefer to keep those two things separated.